pkg/eve/runme.sh

```diff
@@ -100,7 +100,7 @@ create_efi_raw() {
   rm -rf /parts
   ln -s /bits /parts
   dd if=/dev/zero of="$OUTPUT_IMG" seek=$(( $1 * 1024 * 1024 - 1)) bs=1 count=1
-  /make-raw "$OUTPUT_IMG" "$2"
+  /make-raw false "$OUTPUT_IMG" "$2"
```

Comment:
Should it be an option?

Reply:
We currently do not have any option syntax for make-raw, but I start adding that with a -r (for random uuid).

Reply:
I mean for docker run lfedge/eve

Reply:
Fixed